### PR TITLE
Lexer repl

### LINKED ISSUE #97: [lex] Modify defaultStateFn
**Subject**
[lex] Modify defaultStateFn
// TODO: LTE, GTE, spaceStateFn, numberStateFn, identifierStateFn case

**Detail**
- [ ] LTE
- [ ] GTE
- [ ] spaceStateFn
- [ ] numberStateFn
- [ ] identifierStateFn

### PR DESCRIPTION
**Pull request by junbeomlee**
resolved: #97 #98 

details:
1. Add token string function and add cmd lexer
2. Finish to implement defaultStateFn and add test cases

- [x] Test case

### CODE REVIEW

File: cmd/lexer/main.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+	for {
+		fmt.Printf(PROMPT)
+		scanned := scanner.Scan()
+		if !scanned {
+			return
+		}
+
+		line := scanner.Text()
+		if line == EXIT {
```
Comment (boohyunsik) [inline cmd/lexer/main.go:69]: No guide about EXIT command?

Comment (junbeomlee) [reply cmd/lexer/main.go:69]: good comment!
i will add it

**Comment by junbeomlee** [comment]
I added some guide!

**Review by boohyunsik — APPROVED** [review]
Good work!

**Comment by junbeomlee** [comment]
Thx!!

**Pull request merged**